styles/global.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```

Comment:
Can you please add this as a separate commit for tailwind integration with the app? This is independent from storybooks implementation

Reply:
Ok, just saw that this is a separate commit. Resolving this comment